web/src/main/java/com/epam/esm/web/security/userdetails/RegistrationRequest.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.esm.web.security.userdetails;
+
+import java.util.Objects;
+
+public class RegistrationRequest {
```

Comment:
not clear that this class contains user related information, please rename it

Reply:
Renamed to UserRegistrationRequest